src/components/NeedHelpButton/NeedHelpButton.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import { type Meta, type StoryObj } from '@storybook/react'
+import { NeedHelpButton } from './NeedHelpButton'
+
+const meta: Meta<typeof NeedHelpButton> = {
+  title: 'Components/NeedHelpButton',
+  component: NeedHelpButton,
+  tags: ['autodocs']
+}
+
+export default meta
+
+type Story = StoryObj<typeof NeedHelpButton>
+
+export const Primary: Story = {
+  tags: ['autodocs'],
+  args: {
+    title: 'Log In With Google',
+    iconName: 'GoogleIcon',
+    redirectUrl: 'https://www.google.com/'
```

Comment:
I think that these args do not belong to NeedHelpButton component.

Reply:
xd no, they don't, I forgot to delete them